CHANGELOG.md

```diff
@@ -65,6 +65,9 @@
 - Added support fot `pd.json_normalize`.
 - Added support for `GroupBy.pct_change` with `axis=0`, `freq=None`, and `limit=None`.
 - Added support for `DataFrameGroupBy.__iter__` and `SeriesGroupBy.__iter__`.
+- Added partial support for the dataframe interchange protocol method
+  `DataFrame.__dataframe__()`. Snowpark pandas does not yet support columns of
```

Comment:
I thought we support timedelta type? I see the following branch in to_pandas 
```
 if (
                snowpark_pandas_type is not None
                and snowpark_pandas_type == TimedeltaType()
            ):
```

Reply:
is the limitation same as the Snowpark pandas to_pandas limitaiton? if that is the case, maybe just refer to the limitation of to_pandas here

Reply:
sorry, I am confused about this part, i see the implementation is the following 
```
return self.to_pandas().__dataframe__(
            nan_as_null=nan_as_null, allow_copy=allow_copy
        )
```

do you mean the pandasDF.__dataframe__ doesn't support timedelta, what happens when there is a timedelta column after to_pandas(), does it throws an error?
In order to add full support, do you need to go add support in pandas?

Reply:
if that is the case, i don't think we have mention this in the release log, you did a support of DataFrame.__dataframe__(). 
However, mention those limitaitons in the doc under docstrings/dataframe.py, and be clear that those are limitations from https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.__dataframe__.html

Reply:
I disagree, but not strongly enough to engage in a debate. Done.